app/controllers/hourly_rates_controller.rb

```diff
@@ -83,8 +83,14 @@ def update
       return deny_access unless User.current.admin?
     end
 
-    @user.add_rates(@project, permitted_params.user_rates[:new_rate_attributes])
-    @user.set_existing_rates(@project, permitted_params.user_rates[:existing_rate_attributes])
+    if params.include? 'user'
```

Comment:
I think double-quoted strings should only be user when string interpolation is needed.